ci/run-multiple-tests.sh

```diff
@@ -0,0 +1,318 @@
+#! /bin/bash
```

Comment:
Can we try to combine this with #94? I am curious to see whether we can now run tests in parallel.

Reply:
I made a commit a moment ago to update this Bash script to use your new run-test.py program to run individual tests.  We'll see how it goes.